doc-content/enterprise-only/openshift/environment-managed-con.adoc

```diff
@@ -8,10 +8,6 @@ Smart Router is a single endpoint that can receive calls from client application
 
 You must provide a Maven repository with the processes (KJAR files) that you want to deploy on the servers. Your integration process must ensure that the required versions of the processes are uploaded to the Maven repository. You can use {CENTRAL} in a development environment to create the processes and upload them to the Maven repository.
 
-{PRODUCT} includes two base templates for managed environments:
+{PRODUCT} includes the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod.yaml` base templates a for managed environment. It includes two {KIE_SERVER} pods (with database pods), Smart Router in a high-availability configuration, and {CENTRAL} monitoring in a high-availability configuration.
 
-* `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-sit.yaml` is a typical staging environment. It includes two {KIE_SERVER} pods (with database pods), Smart Router, and {CENTRAL} monitoring.
-
-* `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod.yaml` is a typical production environment. It includes two {KIE_SERVER} pods (with database pods), Smart Router in a high-availability configuration, and {CENTRAL} monitoring in a high-availability configuration.
-
-To deploy a managed environment, pick one of the templates, modify it according to your needs if necessary, and then deploy it.
+To deploy a managed environment, deploy this template. You can change the number of replicas of all components when configuring the deployment. If you want to modify the number of independent {KIE_SERVER} pods or to use a different database server (the default is PostgreSQL), you need to modify the template.
```

Comment:
avoid need to, use must IBM: must v
Use “must,” not “have to” or “need to,”
..the default server is or ..the default database is